### ISSUE TITLE
PostMessage in colors

### ISSUE THREAD

**Issue by HardeepAsrani** [Codeinwp/amadeus#57]
Some color fields still reload when you change the value, like Primary Color and Social Section color. @rodica-andronache

**Comment by rodica-andronache** [comment]
@HardeepAsrani will you take care of this or should @stefan-cotitosu fix it? :)

**Comment by HardeepAsrani** [comment]
I'll take care. :)

**Comment by HardeepAsrani** [comment]
Do I need to run grunt tasks here? @rodica-andronache

**Comment by rodica-andronache** [comment]
@HardeepAsrani yes :) I tried updating the grunt files here https://github.com/Codeinwp/amadeus/pull/62

**Comment by rodica-andronache** [comment]
@HardeepAsrani do you still want to do this?

**Comment by HardeepAsrani** [comment]
@rodica-andronache Sure, I didn't get notification of this. Now I check my notification on Github instead of Gmail to make sure I don't miss anything. Sorry for not noticing it.
